E2e tests fail with a typescript compilation error

```
Mon Aug 20 20:08:04 UTC 2018: npm notice created a lockfile as package-lock.json. You should commit this file.
Mon Aug 20 20:08:04 UTC 2018: npm WARN gts@0.8.0 requires a peer of typescript@^2.7.1 but none is installed. You must install peer dependencies yourself.
Mon Aug 20 20:08:04 UTC 2018: 
Mon Aug 20 20:08:04 UTC 2018: + return 0
Mon Aug 20 20:08:04 UTC 2018: + npm run compile
Mon Aug 20 20:08:04 UTC 2018: 
Mon Aug 20 20:08:04 UTC 2018: > @google-cloud/profiler@0.2.0 compile /cloud-profiler-nodejs
Mon Aug 20 20:08:04 UTC 2018: > tsc -p .
Mon Aug 20 20:08:04 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: ts/src/index.ts:273:11 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("/cloud-profiler-nodejs/node_modules/delay/index")' has no compatible call signatures.
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: 273     await delay(profiler.config.localProfilingPeriodMillis / 2);
Mon Aug 20 20:08:08 UTC 2018:               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: ts/src/profilers/time-profiler.ts:47:11 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("/cloud-profiler-nodejs/node_modules/delay/index")' has no compatible call signatures.
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: 47     await delay(durationMillis);
Mon Aug 20 20:08:08 UTC 2018:              ~~~~~~~~~~~~~~~~~~~~~
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: ts/system-test/test-start.ts:92:9 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("/cloud-profiler-nodejs/node_modules/delay/index")' has no compatible call signatures.
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: 92   await delay(30 * 1000);
Mon Aug 20 20:08:08 UTC 2018:            ~~~~~~~~~~~~~~~~
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: ts/test/test-time-profiler.ts:65:16 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("/cloud-profiler-nodejs/node_modules/delay/index")' has no compatible call signatures.
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: 65          await delay(2 * durationMillis);
Mon Aug 20 20:08:08 UTC 2018:                   ~~~~~~~~~~~~~~~~~~~~~~~~~
Mon Aug 20 20:08:08 UTC 2018: 
Mon Aug 20 20:08:08 UTC 2018: npm ERR! code ELIFECYCLE
Mon Aug 20 20:08:08 UTC 2018: npm ERR! errno 1
Mon Aug 20 20:08:08 UTC 2018: npm ERR! @google-cloud/profiler@0.2.0 compile: `tsc -p .`
Mon Aug 20 20:08:08 UTC 2018: npm ERR! Exit status 1
Mon Aug 20 20:08:08 UTC 2018: npm ERR!
Mon Aug 20 20:08:08 UTC 2018: npm ERR! Failed at the @google-cloud/profiler@0.2.0 compile script.
Mon Aug 20 20:08:08 UTC 2018: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Mon Aug 20 20:08:08 UTC 2018: 
```

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
     "@google-cloud/common": "^0.20.3",
     "bindings": "^1.2.1",
     "console-log-level": "^1.4.0",
-    "delay": "^3.0.0",
+    "delay": "~3.0.0",
     "extend": "^3.0.1",
     "gcp-metadata": "^0.7.0",
     "nan": "^2.8.0",

